__tests__/feeds.ts

```diff
@@ -1183,6 +1183,35 @@ describe('query mostUpvotedFeed', () => {
     expect(res.errors).toBeFalsy();
     expect(res.data).toMatchSnapshot();
   });
+
+  it('should not return posts from private squads', async () => {
+    const sourceRepo = con.getRepository(Source);
+    await sourceRepo.update({ id: 'squad' }, { type: SourceType.Squad });
+    const repo = con.getRepository(Post);
+    await repo.update({ id: 'p1' }, { upvotes: 20 });
+    await repo.update({ id: 'p3' }, { upvotes: 15 });
+    await repo.update({ id: 'p2' }, { upvotes: 30, sourceId: 'squad' });
+
+    const res = await client.query(QUERY(30));
+    expect(res.errors).toBeFalsy();
+    expect(res.data).toMatchSnapshot();
```

Comment:
Nothing blocking, but in general snapshots can give us some flakiness on array comparison (as they are not guaranteed order apparently)
For now no action needed, but maybe for future good note to adjust to other evaluations.

Reply:
Yeah, I noticed that the array order was different between when I ran the all the test and the specific test file

Reply:
Yep, array order of snapshot is not guaranteed unfurtionally (you'll find many github issues in Jest repo for this)
https://dailydotdev.slack.com/archives/C02E2C3C13R/p1690297213537409?thread_ts=1690296315.741339&cid=C02E2C3C13R

Reply:
That's quite unfortunate, since snapshot is a great tool.

Reply:
Agreed! 
Hope they'll either support it soon or we can find a good workaround or something.

As mentioned for now don't worry, let's see if they get caught on flakiness at all.